ANSWERS.md

```diff
@@ -1,14 +1,25 @@
 # Q0: Why is this error being thrown?
+No Pokemon controller
 
 # Q1: How are the random Pokemon appearing? What is the common factor between all the possible Pokemon that appear? *
+Seed file generates them. The home controller gets all Pokemons that do not belong to any trainers, and if a trainer is logged in and there exists a Pokemon that does not have a trainer, Pokemon will be displayed.
+None of them has a trainer.
+
 
 # Question 2a: What does the following line do "<%= button_to "Throw a Pokeball!", capture_path(id: @pokemon), :class => "button medium", :method => :patch %>"? Be specific about what "capture_path(id: @pokemon)" is doing. If you're having trouble, look at the Help section in the README.
+capture_path(id: @pokemon) directs the user to the capture method in the pokemon controller and passes in the pokemon's id as a parameter.
 
 # Question 3: What would you name your own Pokemon?
+Hello world.
 
 # Question 4: What did you pass into the redirect_to? If it is a path, what did that path need? If it is not a path, why is it okay not to have a path here?
+I put "/"(home page), and other like trainer_path, capture_path.
+
+If there is not a path then where do we redirect to?
```

Comment:
when you do something like `redirect_to trainer_path(pokemon.trainer_id)` and it takes an argument, it's because `trainer_path` represents a URL with dynamic params, like this: `/users/:id` (`:id` is dynamic: can be 1, 2, 3, etc.)